config/fcos/v1_5_exp/translate.go

```diff
@@ -362,3 +362,28 @@ func buildGrubConfig(gb Grub) string {
 	superUserCmd := fmt.Sprintf("set superusers=\"%s\"\n", strings.Join(allUsers, " "))
 	return "# Generated by Butane\n\n" + superUserCmd + strings.Join(cmds, "\n") + "\n"
 }
+
+func arePartitionEqual(p1, p2 types.Partition) bool {
+    return p1.Label == p2.Label && p1.Number == p2.Number
+}
+
+
+func checkPartitionTable(disk string, wipeTable bool, partitions []types.Partition) error {
+   // if wipetable is true do nothing
+	if wipeTable {
+        return nil
+    }
+   // check partitions labels and numbers and see if there are duplicates
+
+   // problably a faster way to do this... 
+    for i, p1 := range partitions {
+        for j := i + 1; j < len(partitions); j++ {
+            p2 := partitions[j]
+            if arePartitionEqual(p1, p2) {
```

Comment:
This doesn't match the logic described in #377, and in fact won't warn on the example given there.  How did you arrive at this approach?

Reply:
Oh I thought the labels and numbers of each partition could not be the same, ill change it so it checks if the number of the partition table is 0 and wipe_tables is false and warn if that's the case

Reply:
There are already checks for overlapping labels/numbers, but confusingly they're [in a different repo](https://github.com/coreos/ignition/blob/5c31ef8eb3670e24bc4a02dc465d82107c2913a1/config/v3_5_experimental/types/disk.go#L29-L49).  (Ignition has its own config validation, and Butane runs it after translating the config.)